man/DataFrame_class.Rd

```diff
@@ -116,9 +116,7 @@ withr::with_timezone(
   \}
 )
 #> <error: in to_r: ComputeError(ErrString("datetime '2020-03-08 02:00:00' is non-existent in time zone 'America/New_York'. You may be able to use `non_existent='null'` to return `null` in this case.")) When calling: devtools::document()>
-}\if{html}{\out{</div>}}
```

Comment:
I think they are unrelated changes.
@etiennebacher Maybe you haven't updated R packages? (maybe knitr)

Reply:
I have everything up to date. Those were added in #1195, I guess by mistake

Reply:
Hmmm, how about the pandoc version?